mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -146,6 +151,55 @@ class Word2VecModel private[ml] (
     wordVectors: feature.Word2VecModel)
   extends Model[Word2VecModel] with Word2VecBase {
 
+
+  /**
+   * Return a map of every word to its Vector representation.
+   */
+  val getVectors: Map[String, Array[Float]] = wordVectors.getVectors
```

Comment:
@mengxr I would like to return this as a dataframe with each row having two fields, a string and a float array. But I'm unable to understand how to access the SparkContext here.